core/src/test/java/com/crawljax/core/CandidateElementExtractorTest.java

```diff
@@ -179,6 +179,31 @@ public void whenFollowExternalUrlDoFollow() throws IOException, URISyntaxExcepti
 		assertThat(extract, hasSize(3));
 	}
 
+	@Test
+	public void testExtractContentSpecificFilenameProblem() throws Exception {
+		RunWithWebServer TEST_SITE_SERVER = new RunWithWebServer("/site/element-extractor");
```

Comment:
The variable should be in camel case, although this test could avoid using the server by using `extractFromTestFile` (tweaked to a custom file path).